src/locales/en.json

```diff
@@ -56,30 +58,6 @@
   "error.incompleteForm": "Please fill in the form completely!",
   "error.retry": "Please try again!",
   "shell.link": "here",
-  "portfolio.details.editPortfolio": "Edit Portfolio",
-  "portfolio.details.summaryHeader": "Summary",
-  "portfolio.details.dialogHeader": "Here you can adjust the positions of your portfolio",
-  "portfolio.details.discardChanges": "Discard changes",
-  "portfolio.details.saveChanges": "Save changes",
-  "portfolio.details.score": "Score",
-  "portfolio.details.day7": "7 Day",
-  "portfolio.details.year": "1 Year",
-  "portfolio.details.totalValue": "Total Value",
-  "portfolio.details.positions": "Positions",
-  "portfolio.details.countries": "Countries",
-  "portfolio.details.industries": "Industries",
-  "portfolio.details.currencies": "Currencies",
-  "portfolio.details.positionsTitle": "Positions",
-  "portfolio.details.viewMore": "view more",
-  "portfolio.details.risk": "Risk Analysis",
-  "portfolio.details.segments": "Segments",
-  "portfolio.details.currency": "Currency",
-  "portfolio.details.keyfigures": "Key Figures",
-  "portfolio.details.dividends": "Dividends",
-  "portfolio.details.divYield": "Dividend Yield",
-  "portfolio.details.payout": "Dividend Payout Ratio",
-  "portfolio.details.nextDate": "Next Payout Date",
-  "portfolio.details.analyst": "Analysts",
```

Comment:
Where all of these duplicated or why does it show this as deleting them all?

Reply:
Yes. The duplication happened [here](https://github.com/ProjectMilou/frontend/pull/122/files#diff-b1791c7a47875ce85f2f1e57ef6c25a9b993660f9bcc840d8f21daac6161b5a0R56-R79).